app/views/projects/show.html.haml

```diff
@@ -59,7 +59,7 @@
             %span.light Created on
             #{@project.created_at.stamp('Aug 22, 2013')}
           %p
-            %span.light Owned by
+            %span.light Owned by #{@project.group ? "the" : nil}
             - if @project.group
```

Comment:
I wanted to continue having `%span.light` applied to the text. Doing it below removed it and didn't make things look very good.